koku/masu/processor/ocp/ocp_cost_model_cost_updater.py

```diff
@@ -42,6 +42,7 @@ def __init__(self, schema, provider):
         self._cluster_id = get_cluster_id_from_provider(self._provider_uuid)
         self._cluster_alias = get_cluster_alias_from_cluster_id(self._cluster_id)
         with CostModelDBAccessor(self._schema, self._provider_uuid) as cost_model_accessor:
+            self._cost_model_found = cost_model_accessor._cost_model
```

Comment:
This variable name isn't really accurate. Let's drop the `_found`:
```suggestion
            self._cost_model = cost_model_accessor._cost_model
```